src/environmentd/tests/sql.rs

```diff
@@ -1461,6 +1461,39 @@ fn test_internal_ports() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+// Test that trying to alter an invalid system param returns an error.
```

Comment:
Why can't this go into test/sqllogictest? Needs a specific user?

Reply:
Yeah it needs a specific user and port, which I don't think is possible in test/sqllogictest without changing the framework.